utils/http_test.go

```diff
@@ -23,6 +23,14 @@ func Test_GetMIME(t *testing.T) {
 
 	res = GetMIME("unknown")
 	AssertEqual(t, MIMEOctetStream, res)
+
+	if err := mime.AddExtensionType(".mjs", "application/javascript"); err == nil {
+		res = GetMIME(".mjs")
+		AssertEqual(t, "application/javascript", res)
+	} else {
+		t.Logf("failed to add extension type, err: %v", err)
```

Comment:
This must also never fail -- use `AssertEqual(t, nil, err)` here instead.

Reply:
@leonklingele thanks for reviewing. done